Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/Destination.cs

```diff
@@ -1,79 +1,78 @@
-        /// </summary>
-        /// <example>utf-64</example>
-        [UIHint(nameof(FileNameEncoding), "", FileEncoding.Other)]
-        public string FileNameEncodingInString { get; set; }
+    /// <summary>
+    /// File encoding to be used. A partial list of possible encodings: https://en.wikipedia.org/wiki/Windows_code_page#List.
+    /// </summary>
+    /// <example>utf-64</example>
+    [UIHint(nameof(FileNameEncoding), "", FileEncoding.Other)]
+    public string FileNameEncodingInString { get; set; }
 
-        /// <summary>
-        /// Operation to determine what to do if destination file exists.
-        /// </summary>
-        /// <example>DestinationAction.Error</example>
-        [DefaultValue(DestinationAction.Error)]
-        public DestinationAction Action { get; set; }
+    /// <summary>
+    /// Operation to determine what to do if destination file exists.
+    /// </summary>
+    /// <example>DestinationAction.Error</example>
+    [DefaultValue(DestinationAction.Error)]
+    public DestinationAction Action { get; set; }
 
-        /// <summary>
-        /// Encoding for the appending content. By selecting 'Other' you can use any encoding.
-        /// </summary>
-        /// <example>FileEncoding.ANSI</example>
-        [DefaultValue(FileEncoding.ANSI)]
-        [UIHint(nameof(Action), "", DestinationAction.Append)]
-        public FileEncoding FileContentEncoding { get; set; }
+    /// <summary>
+    /// Encoding for the appending content. By selecting 'Other' you can use any encoding.
+    /// </summary>
+    /// <example>FileEncoding.ANSI</example>
+    [DefaultValue(FileEncoding.ANSI)]
+    [UIHint(nameof(Action), "", DestinationAction.Append)]
+    public FileEncoding FileContentEncoding { get; set; }
 
-        /// <summary>
-        /// File encoding to be used. A partial list of possible encodings: https://en.wikipedia.org/wiki/Windows_code_page#List.
-        /// </summary>
-        /// <example>utf-64</example>
-        [UIHint(nameof(FileContentEncoding), "", FileEncoding.Other)]
-        public string FileContentEncodingInString { get; set; }
+    /// <summary>
+    /// File encoding to be used. A partial list of possible encodings: https://en.wikipedia.org/wiki/Windows_code_page#List.
+    /// </summary>
+    /// <example>utf-64</example>
+    [UIHint(nameof(FileContentEncoding), "", FileEncoding.Other)]
+    public string FileContentEncodingInString { get; set; }
 
-        /// <summary>
-        /// Additional option for UTF-8 encoding to enable bom.
-        /// </summary>
-        /// <example>true</example>
-        [UIHint(nameof(FileContentEncoding), "", FileEncoding.UTF8)]
-        public bool EnableBomForContent { get; set; }
-    }
+    /// <summary>
+    /// Additional option for UTF-8 encoding to enable bom.
+    /// </summary>
+    /// <example>true</example>
+    [UIHint(nameof(FileContentEncoding), "", FileEncoding.UTF8)]
+    public bool EnableBomForContent { get; set; }
```

Comment:
You are not using this property anywhere, so it can be removed.

Reply:
Same thing here. This is now fixed.

Reply:
You are still not using the property, so it can be removed.